manifests/profile/www_lib/vhosts/med.pp

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) 2020 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# nebula::profile::www_lib::vhosts::med
+#
+# Middle English Dictionary virtual host
+#
+# @example
+#   include nebula::profile::www_lib::vhosts::med
+class nebula::profile::www_lib::vhosts::med (
+  Array[String] $ip_restrict,
+  String $docroot = '/hydra/dromedary-production/current/public',
+) {
+  $servername = 'med.lib.umich.edu'
+
+  file { "${apache::params::logroot}/med":
```

Comment:
This looks like it might be out of sync with the vhost below.

Reply:
Good catch. Thanks.